tensorflow/compiler/tf2tensorrt/utils/trt_shape_optimization_profiles_test.cc

```diff
@@ -123,14 +123,14 @@ class TrtShapeOptimizationProfileTest : public ::testing::Test {
 };
 
 TEST_F(TrtShapeOptimizationProfileTest, Static) {
-  // Network with static input shape
+  // Network with static input shape.
```

Comment:
This last line check EXPECT_EQ(-1, profile.GetProfileNumber(shape_vec)) in this test fail when I ran the test. I couldn't put a comment next to that line, so putting it here.